src/emu/debug/debughlp.cpp

```diff
@@ -21,6 +21,8 @@
 #define CONSOLE_HISTORY     (10000)
 #define CONSOLE_LINE_CHARS  (100)
 
+static const std::string html_preopen = "<pre>";
+static const std::string html_preclose = "</pre>";
```

Comment:
Why are these static string objects?  It increases static initialisation overhead on all runs even if the HTML output function is never used.

Reply:
They are strings to allow easy length measuring. Is the fact that they are static the only problem?